src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.cpp

```diff
@@ -127,6 +127,31 @@ tnsr::abb<DataType, Dim, Frame> compute_derivatives_of_spacetime_metric(
   return spacetime_deriv_spacetime_metric;
 }
 
+template <size_t SpatialDim, typename Frame, typename DataType>
+tnsr::a<DataType, SpatialDim, Frame> compute_spacetime_normal_one_form(
+    const Scalar<DataType>& lapse) noexcept {
+  tnsr::a<DataType, SpatialDim, Frame> normal_one_form{};
+  get<0>(normal_one_form) = -get<>(lapse);
+  for (size_t i = 0; i < SpatialDim; ++i) {
+    normal_one_form.get(i + 1) = make_with_value<DataType>(get<>(lapse), 0.);
+  }
+  return normal_one_form;
+}
+
+template <size_t SpatialDim, typename Frame, typename DataType>
+tnsr::A<DataType, SpatialDim, Frame> compute_spacetime_normal_vector(
+    const Scalar<DataType>& lapse,
+    const tnsr::I<DataType, SpatialDim, Frame>& shift) noexcept {
+  tnsr::A<DataType, SpatialDim, Frame> T_I{};
```

Comment:
What does this variable name mean?

Reply:
Whoops, this was from spec where the variable name was trying to mimic what the variable would look like in math, so the normal is denoted T and I is and upper i index. I don't like it so will change.